crates/ironrdp-cliprdr/src/lib.rs

```diff
@@ -47,7 +47,7 @@ enum CliprdrState {
 /// CLIPRDR static virtual channel client endpoint implementation
 #[derive(Debug)]
 pub struct Cliprdr {
-    backend: Box<dyn CliprdrBackend>,
+    pub backend: Box<dyn CliprdrBackend>,
```

Comment:
Let's make getter for this instead

Reply:
What about exposing it with methods like these?

```rust
fn downcast_backend<T: Any>(&self) -> Option<&T> {
    self.backend.as_any().downcast::<T>()
}

fn downcast_backend_mut<T: Any>(&mut self) -> Option<&mut T> {
    self.backend.as_any_mut().downcast_mut::<T>()
}
```

Reply:
@pacmancoder sounds good.

@CBenoit I tried that initially but it breaks object safety (due to generics in trait methods) which is required for having `dyn CliprdrBackend`

Reply:
@CBenoit ah I see, adding those to `Cliprdr` itself 👍.